client/src/styles/global.scss

```diff
@@ -17,4 +17,11 @@
 
 .j40-two-column-icons-spacing {
   padding-right: 1em;
+  width: 2.3em;
```

Comment:
I will need to merge with my pending PR. (That will teach me to sit on a merge) 😆

Reply:
ah, this originally came from a merge with `main` actually so it should merge cleanly